packages/create-next-app/create-app.ts

```diff
@@ -128,6 +128,8 @@ export async function createApp({
 
   process.chdir(root)
 
+  const hasPackageJSON = await existsInRepo(`${example}/package.json`)
```

Comment:
Instead of checking if a `package.json` exists in the remote resource should we check if there's a `package.json` locally after the download step?